check.py

```diff
@@ -365,17 +365,29 @@ def run_unittest():
     ('gcc', run_gcc_tests),
     ('unit', run_unittest),
     ('binaryenjs', binaryenjs.test_binaryen_js),
+    ('binaryenjs_wasm', binaryenjs.test_binaryen_wasm),
 ])
 
 
 # Run all the tests
 def main():
+    all_suites = TEST_SUITES.keys()
+    skip_by_default = ['binaryenjs', 'binaryenjs_wasm']
```

Comment:
Nice, it has always been a small annoyance that the JS tests run automatically even when my JS build is out of date. Can we add similar logic to `auto_update_tests.py`, too?